README.md

```diff
@@ -1,3 +1,84 @@
-# CTD Omega Services
+# Pan-Archival Catalogue Services
```

Comment:
How about a description of the available services (currently just 'echo')?

Reply:
Added.